testInput2.txt

```diff
@@ -0,0 +1,4 @@
+gps_timestamp lat lng alt roll_deg pitch_deg yaw_deg
```

Comment:
Seems this file is unrelated to your current code, you can delete it.

Reply:
i planning on testing by write and reading this file to database.